src/platform/silabs/EndpointQueueFilter.cpp

```diff
@@ -0,0 +1,108 @@
+#include "EndpointQueueFilter.h"
+#include <string.h>
+#include <support/CodeUtils.h>
+#include <support/logging/CHIPLogging.h>
+#include <string_view>
+
+namespace chip {
+namespace Inet {
+
+using FilterOutcome = EndpointQueueFilter::FilterOutcome;
+
+namespace {
+
+bool IsValidMdnsHostName(uint8_t * hostName)
```

Comment:
using a CharSpan. This will ensure sizing is valid and no assumptions can be broken later.

Reply:
Modified to use CharSpan